frontend/src/pages/UserPage.vue

```diff
@@ -0,0 +1,81 @@
+<script lang="ts" setup>
+import { useUserStore } from '@/store/user';
+import Dropdown from '@/components/Dropdown.vue';
+import DropdownItem from '@/components/DropdownItem.vue';
+import NameLink from '@/components/NameLink.vue';
+import MainButton from '@/components/MainButton.vue';
+import router from '@/routes/router';
+import { Course, Session } from '@gym-manager/models';
+import { ref } from 'vue';
+
+const store = useUserStore();
+
+const user = store.client.getUserDetails;
```

Comment:
Note for @eutampieri : this getter shouldn’t include the "get" part. Check the interface